lxd/images.go

```diff
@@ -106,7 +106,7 @@ func unpack(file string, path string) error {
 	if err != nil {
 		shared.Debugf("Unpacking failed")
 		shared.Debugf(string(output))
-		return err
+		return fmt.Errorf("Unpack failed, %s.", err)
```

Comment:
Here it might be nice to include the output, since I think `err` is just some garbage golang gives us about the exit code of the command.